example/src/main/java/link/magic/demo/magic/MCFragment.kt

```diff
@@ -114,4 +116,10 @@ class MCFragment: Fragment() {
             }
         }
     }
+
+    override fun onMagicEvent(eventType: String, data: String) {
```

Comment:
Registering the event listener on the page is a DX friction. but given that it's Android, I don't think we have other good options, isn't it?